walg.c

```diff
@@ -91,7 +91,7 @@ usenext(Wal *w)
 static int
 ratio(Wal *w)
 {
-    int n, d;
+    int64 n, d;
 
     d = w->alive + w->resv;
     n = w->nfile*w->filesize - d;
```

Comment:
I think filesize will also have to be 64 bits, and this expression should convert nfile to int64 explicitly, to make sure the multiply is done in 64 bits.